api/test-requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest~=4.6.7 # needed for python 2.7+3.4
-pytest-cov>=2.8.1
-pytest-randomly==1.2.3 # needed for python 2.7+3.4
-Flask-Testing==0.8.0
+pytest==7.3.1
+pytest-cov==4.0.0
+pytest-randomly==3.12.0 
+Flask-Testing==0.8.1
```

Comment:
was getting errors about dependencies when installing as well. why does the pytest version need to be considered for python 2.7?